### ISSUE TITLE
SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data

### ISSUE THREAD

**Issue by bradcypert** [PyreStudios/steward#25]
When running the following code, I get a `SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data` error. The issue is likely the writeResponse method trying to infer the content type. This instead should render this with content type text/plain

```dart
  router.get('/version', (Request request) async {
    return Response.Ok('Version 1.0');
  });
```

**Comment by bradcypert** [comment]
I am looking into this.

**Comment by bradcypert** [comment]
Fixed in v0.2.8
